<ui version="4.0" >
 <class>eVisDatabaseLayerFieldSelectionGuiBase</class>
 <widget class="QDialog" name="eVisDatabaseLayerFieldSelectionGuiBase" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Database File Selection</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resources/evis.qrc" >
    <normaloff>:/evis/eVisDatabaseConnection.png</normaloff>:/evis/eVisDatabaseConnection.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="3" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QComboBox" name="cboxYCoordinate" >
     <property name="toolTip" >
      <string/>
     </property>
     <property name="whatsThis" >
      <string>The name of the field that contains the Y coordinate of the points.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QComboBox" name="cboxXCoordinate" >
     <property name="toolTip" >
      <string/>
     </property>
     <property name="statusTip" >
      <string/>
     </property>
     <property name="whatsThis" >
      <string>The name of the field that contains the X coordinate of the points.</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="leLayerName" >
     <property name="toolTip" >
      <string/>
     </property>
     <property name="whatsThis" >
      <string>Enter the name for the new layer that will be created and displayed in QGIS.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Y Coordinate</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>X Coordinate</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Name of New Layer</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/evis.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>eVisDatabaseLayerFieldSelectionGuiBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>eVisDatabaseLayerFieldSelectionGuiBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
